src/draw/lv_image_decoder.c

```diff
@@ -137,6 +148,11 @@ lv_result_t lv_image_decoder_open(lv_image_decoder_dsc_t * dsc, const void * src
         if(dsc->header.stride == 0) dsc->header.stride = img_width_to_stride(&dsc->header);
 
         dsc->decoder = decoder;
+        dsc->cache = img_cache_p;
```

Comment:
It's a global image cache, thus no need to store cache instance in every decoded image.

Reply:
In this way, I don't need to include lv_global and define the image_cache macro again in each decoder.